src/lib/embed.js

```diff
@@ -154,8 +155,9 @@ docReady(function() {
             return 0;
         }
 
-        // only load jQuery if not already included in page.
-        if (!(j = window.jQuery) || compareVersionNumbers(version, j.fn.jquery) || callback(j, scriptUri, absScriptUri, loaded)) {
+        // load jQuery if not already included in page or
+        // if the version we want (1.10.2) is greater than what's available
+        if (!(j = window.jQuery) || isPositiveInteger(compareVersionNumbers(version, j.fn.jquery)) || callback(j, scriptUri, absScriptUri, loaded)) {
```

Comment:
I would find this code easier to read if the function was more descriptive. e.g. 
```javascript
if(!correctVersionOfJqueryLoaded()) {
  dynamicallyLoadJquery();
}
```